src/components/wallets/WalletAccordion.tsx

```diff
@@ -0,0 +1,327 @@
+          </SummaryRight>
+        </Summary>
+
+        <Details>
+          <DetailSummary>
+            <DetailSummaryBlock>
+              <DetailSummaryLine>
+                <Typography color="grey500" marginRight={theme.spacing(1)}>
+                  {translate('text_62da6ec24a8e24e44f812876')}
+                </Typography>
+                <TooltipIcon
+                  placement="bottom-start"
+                  title={translate('text_62da6db136909f52c2704c40', {
+                    date: DateTime.fromISO(wallet.lastBalanceSyncAt).toFormat('LLL. dd yyyy'),
+                  })}
+                >
+                  <Icon name="info-circle" />
+                </TooltipIcon>
+              </DetailSummaryLine>
+              <DetailSummaryLine>
+                <Typography color="grey700" variant="subhead">
+                  {creditAmountUnit}
+                </Typography>
+                {creditAmountCents && (
+                  <Typography color="grey700" variant="bodyHl">
+                    .{creditAmountCents}
+                  </Typography>
+                )}
+                <Typography color="grey700" variant="bodyHl" marginLeft={theme.spacing(1)}>
+                  {translate(
+                    'text_62da6ec24a8e24e44f81287a',
+                    {},
+                    Math.max(Number(creditAmountUnit) || Number(creditAmountCents))
+                  )}
+                </Typography>
+              </DetailSummaryLine>
+              <DetailSummaryLine>
+                <Typography color="grey600" marginRight={theme.spacing(1)} variant="caption">
+                  {wallet.currency}
+                </Typography>
+                <Typography color="grey600" variant="caption">
+                  {wallet.creditsBalance}
+                </Typography>
+              </DetailSummaryLine>
+            </DetailSummaryBlock>
+
+            <DetailSummaryBlock>
+              <DetailSummaryLine>
+                <Typography color="grey500" marginRight={theme.spacing(1)}>
+                  {translate('text_62da6ec24a8e24e44f812880')}
+                </Typography>
+                <TooltipIcon
+                  placement="bottom-start"
+                  title={translate('text_62da6db136909f52c2704c40', {
+                    date: DateTime.fromISO(wallet.lastConsumedCreditAt).toFormat('LLL. dd yyyy'),
+                  })}
+                >
+                  <Icon name="info-circle" />
+                </TooltipIcon>
+              </DetailSummaryLine>
+              <DetailSummaryLine>
+                <Typography color="grey700" variant="subhead">
+                  {consumerCreditUnit}
+                </Typography>
+                {creditAmountCents && (
+                  <Typography color="grey700" variant="bodyHl">
+                    .{consumerCreditCents}
+                  </Typography>
+                )}
+                <Typography color="grey700" variant="bodyHl" marginLeft={theme.spacing(1)}>
```

Comment:
No inline styling. If you want you can remove it from the Typography type.
Also, margin should always be on the right side or bottom of the element.
This is a convention decided with Mike to match the designs.

Reply:
Do not use the styling inline (for margin, padding etc) except for the skeletons.
Reason is that to maintainers it get really annoying if we use both options to find where it's styled.

Reply:
Updated